RetailCoder.VBE/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -361,9 +359,9 @@ private void ExecuteQuickFixInModuleCommand(object parameter)
                 .ToList();
 
             var items = filteredResults
-                .Where(result => !(result is AggregateInspectionResult))
+                .Where(result => result.GetType().Name != "AggregateInspectionResult")
```

Comment:
@Vogel612 well spotted - I forgot to get back to that hack after moving `AggregateInspectionResult` to the `Rubberduck.UI.Inspections` namespace, which makes the aggregate results nothing more than a presentation concern (which it really is after all) - I disagree that individual inspections should be responsible for aggregating their own results.